src/main/java/games/strategy/triplea/ui/ObjectivePanel.java

```diff
@@ -613,30 +613,29 @@ public String getProperty(final String objectiveKey) {
   }
 
   public String getProperty(final String objectiveKey, final String defaultValue) {
-    return m_properties.getProperty(objectiveKey, defaultValue);
+    return properties.getProperty(objectiveKey, defaultValue);
   }
 
   public Set<Entry<Object, Object>> entrySet() {
-    return m_properties.entrySet();
+    return properties.entrySet();
   }
 }
 
 
 class ObjectivePanelDummyDelegateBridge implements IDelegateBridge {
```

Comment:
I believe this is just an in-memory stub used within the UI.  There's no actual delegate involved, so there should be no compatibility issues.